### ISSUE TITLE
False positive with SC2016

### ISSUE THREAD

**Issue by enoch85** [koalaman/shellcheck#1601]
#### For bugs
- Rule Id: SC2016
- My shellcheck version: Travis latest.
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [X] I tried on shellcheck.net and verified that this **gives no errors** on that site.

![Screenshot from 2019-06-06 19:10:19](https://user-images.githubusercontent.com/4511254/59052191-cb5b4280-888e-11e9-86c7-d0ed29fbb384.png)

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:
![Screenshot from 2019-06-06 19:11:02](https://user-images.githubusercontent.com/4511254/59052215-dd3ce580-888e-11e9-9590-b60212b85437.png)

```sh
#!/bin/bash

# Remove old watchtower if existing
if does_this_docker_exist v2tec/watchtower
then
    # Get Env values
    docker inspect -f '{{range $index, $value := .Config.Env}}{{$value}}{{println}}{{end}}' watchtower > env.list

    # Remove empty lines
    sed -i '/^[[:space:]]*$/d' env.list

    # Get Cmd values
    CmdDocker=$(docker inspect --format='{{.Config.Cmd}}' watchtower | cut -d "]" -f 1 | cut -d "[" -f 2;)

    # Check if env.list is empty and run the docker accordingly
    if [ -s env.list ]
    then
        docker_prune_this v2tec/watchtower
        docker run -d --restart=unless-stopped --name watchtower -v /var/run/docker.sock:/var/run/docker.sock --env-file ./env.list containrrr/watchtower "$CmdDocker"
        rm -f env.list
    else
        docker_prune_this v2tec/watchtower
        docker run -d --restart=unless-stopped --name watchtower -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower "$CmdDocker"
    fi
fi

```

#### Here's what shellcheck currently says:
`^-- SC2016: Expressions don't expand in single quotes, use double quotes for that.` when running travis.

#### Here's what I wanted or expected to see:
That Travis exited with 0 as it works on shellcheck.net

**Comment by enoch85** [comment]
This can be closed. https://github.com/nextcloud/vm/pull/844/commits/0af38fe9a6d37a68f9d32459e521e0c2cc94b444

From https://github.com/koalaman/shellcheck/issues/375#issuecomment-102442997
